### PR TITLE
EVG-6548: add menu options in app bar

### PR DESCRIPTION
**Pull request by dominoweir**

### CODE REVIEW

File: src/components/app/App.tsx [hunk 1]
```diff
@@ -119,13 +168,13 @@ export class Evergreen extends React.Component<Props, State> {
     return app;
   }
 
-  // private openMenu = (event: React.MouseEvent<HTMLElement>) => {
```
Comment (john-m-liu) [inline src/components/app/App.tsx]: Let's keep these comments too

File: src/components/app/App.tsx [hunk 2]
```diff
@@ -75,9 +75,58 @@ export class Evergreen extends React.Component<Props, State> {
                 <AppBar className="app-bar" >
                   <Toolbar>
                     <img src={EvergreenIcon} className="app-icon" />
-                    <Typography variant="h5" color="inherit" noWrap={true}>
-                      Evergreen
+                    <Link href="https://evergreen.mongodb.com/waterfall" underline="none">
```
[the remark this answers is not in the document]
Comment (dominoweir) [reply src/components/app/App.tsx]: Fixed

File: src/components/app/App.tsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ export class Evergreen extends React.Component<Props, State> {
     const config = () => <ConfigDrop updateClientConfig={this.updateConfig} onLoadFinished={null} />
     const build = () => <BuildView client={this.state.APIClient} />
     let ErrorHandler: any;
-    // const menuOpen = Boolean(this.state.MenuAnchor);
```
[the remark this answers is not in the document]
Comment (dominoweir) [reply src/components/app/App.tsx]: Fixed

File: src/components/app/App.tsx (hunk 1 above)
Comment (dominoweir) [reply src/components/app/App.tsx]: Fixed

**Pull request merged**